src/constants.js

```diff
@@ -2,5 +2,9 @@ export default {
     MESSAGE_ERROR: 0,
     MESSAGE_STATUS: 1,
     MESSAGE_SUCCESS: 2,
-    MESSAGE_WARNING: 3
+    MESSAGE_WARNING: 3,
+    copy: {
+        PAYMENT_REQUEST_DEFAULT: "Thanks, your payment request has been sent. Please take your vouchers to your market representative for them to be sent off.",
+        PAYMENT_REQUEST_ERROR: "There was a problem with your payment request, please try again later.",
```

Comment:
See above. Also, i remember Katja saying in scoping that she'd prefer if we used the server-passed error for error messages rather than  having the UI layer reinterpret them. There may be another card for that.